Tools/autotest/rover.py

```diff
@@ -303,6 +303,41 @@ def test_sprayer(self):
         if ex:
             raise ex
 
+    def drive_max_rcin(self, timeout=30):
+        """Test max RC inputs"""
+        self.context_push()
+        ex = None
+
+        try:
+            self.progress("Testing max RC inputs")
+            self.wait_mode("MANUAL")
+
+            self.wait_ready_to_arm()
+            self.arm_vehicle()
+
+            self.set_rc(3, 2000)
+            self.set_rc(1, 1000)
+
+            tstart = self.get_sim_time()
+            while self.get_sim_time() < tstart + timeout:
+                m = self.mav.recv_match(type='VFR_HUD', blocking=True)
```

Comment:
Need to add a timeout here.  Note that `m` can then come back as `None` and you have to handle that.

If you don't do this then the suite can hang for prolonged periods of time and everything past this will fail.

Reply:
Timeout added